### PR TITLE
refactor(esx_menu_default/dialog): Optimalization

### PR DESCRIPTION
**Pull request by itIsMaku**
Optimized checking keys and actions.
Removed while true do loops.

### CODE REVIEW

File: [esx]/esx_menu_default/client/main.lua [hunk 1]
```diff
@@ -1,93 +1,116 @@
-local GUI, MenuType = {}, 'default'
+local GUI, MenuType = {}, "default"
```
Comment (mcNuggets1) [inline [esx]/esx_menu_default/client/main.lua:1]: very optimised

Comment (itIsMaku) [reply [esx]/esx_menu_default/client/main.lua:1]: You should take a look on commits. As you can see, there is a commit with reformatting.

Comment (mcNuggets1) [reply [esx]/esx_menu_default/client/main.lua:1]: But why in a different stlye than the rest of ESX Legacy, doesnt make sense

**Review by GRZ-D3V — APPROVED** [review]

**Comment by Benzo00** [comment]
Proper title is needed, check issues here on legacy GitHub to see pr guidelines

**Pull request closed without merging**